.replit

```diff
@@ -1,6 +1,6 @@
 run = "npm run test"
 hidden = [".config", "package-lock.json"]
-modules = ["nodejs-20:v8-20230920-bd784b9"]
+modules = ["nodejs-with-prybar-18:v26-20240401-269b323"]
```

Comment:
Why did we downgrade to this module of all things? Prybar is deprecated.

Reply:
i was actually just trying to match the semaphore environment which is node 18, is there another node 18 environment i should use?

Reply:
fixed